scripts/travis/Jenkinsfile

```diff
@@ -0,0 +1,43 @@
+tmatrix = []
+def buildClosures(arg) {
```

Comment:
Can you provide some inline documentation here as to what all this code does.  It's pretty opaque otherwise.